app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -353,7 +358,7 @@ public boolean onMenuItemClick(MenuItem item) {
                 try {
                     Intent intent = new Intent(Intent.ACTION_OPEN_DOCUMENT_TREE);
                     intent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
-                    startActivityForResult(intent, REQUEST_CODE_ADD_LOCAL_FOLDER);
+                    addLocalFolderLauncher.launch(intent);
```

Comment:
Looks like this should use `ActivityResultContracts.OpenDocumentTree` instead of `ActivityResultContracts.StartActivityForResult` with a manual intent. Same with some of the other uses of the new `ActivityResultLauncher`

Reply:
I'll look into this :+1: